common/lib/xmodule/xmodule/js/src/html/edit.js

```diff
@@ -1406,7 +1410,7 @@
       if (this.editor_choice === 'visual') {
         visualEditor = this.getVisualEditor();
         raw_content = visualEditor.getContent({
-          format: "raw",
+          format: "text",
```

Comment:
[TinyMCE 5 uses](https://www.tiny.cloud/docs/api/tinymce/tinymce.editor/#getcontent) the value `text` instead of `raw` for the `format`.